### PR TITLE
OZ-587: Added DOB to Quotation and Partner entity

### PR DESCRIPTION
**Pull request by VaishSiddharth**
JIRA: https://mekomsolutions.atlassian.net/browse/OZ-587

### CODE REVIEW

File: odoo-openmrs/src/test/java/com/ozonehis/eip/odoo/openmrs/mapper/odoo/PartnerMapperTest.java [hunk 1]
```diff
@@ -126,6 +128,7 @@ private static Patient getPatient() {
         Patient patient = new Patient();
         patient.setId("123");
         patient.setActive(true);
+        patient.setBirthDate(new Date(883414538000L));
```
Comment (corneliouzbett) [inline odoo-openmrs/src/test/java/com/ozonehis/eip/odoo/openmrs/mapper/odoo/PartnerMapperTest.java:131]: I don't like setting the date with a long value, as it's not easy to read. Maybe try using `Calendar` or `SimpleDateFormat`?

File: odoo-openmrs/src/main/java/com/ozonehis/eip/odoo/openmrs/client/OdooUtils.java [hunk 2]
```diff
@@ -84,4 +87,17 @@ private Object convertValueToType(Object value, Class<?> targetType) {
         }
         throw new IllegalArgumentException("Unsupported type: " + targetType.getName());
     }
+
+    public static String convertEEE_MMM_ddDateToOdooFormat(String date) {
```
Comment (corneliouzbett) [inline odoo-openmrs/src/main/java/com/ozonehis/eip/odoo/openmrs/client/OdooUtils.java:91]: It's anti-pattern to use underscores in the Java method names
```suggestion
    public static String convertToOdooDateFormat(String date) {
```

**Review by corneliouzbett — COMMENTED** [review]
Left few comments otherwise this already LGTM

File: odoo-openmrs/src/main/java/com/ozonehis/eip/odoo/openmrs/client/OdooUtils.java (hunk 2 above)
Comment (VaishSiddharth) [reply odoo-openmrs/src/main/java/com/ozonehis/eip/odoo/openmrs/client/OdooUtils.java:91]: Wanted to keep this method name generic by specifying the date format and not saying `OdooDateFormat`
Will rename it to `convertEEEMMMddDateToOdooFormat`

File: odoo-openmrs/src/test/java/com/ozonehis/eip/odoo/openmrs/client/OdooUtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2021, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.eip.odoo.openmrs.client;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (corneliouzbett) [inline odoo-openmrs/src/test/java/com/ozonehis/eip/odoo/openmrs/client/OdooUtilsTest.java:10]: Avoid wildcards imports

Comment (VaishSiddharth) [reply odoo-openmrs/src/test/java/com/ozonehis/eip/odoo/openmrs/client/OdooUtilsTest.java]: Thanks for pointing out. Something weird with my IDE it doesn't save this setting.

**Review by corneliouzbett — APPROVED** [review]
LGTM!

**Pull request merged**